beta/examples/tensorflow/common/argparser.py

```diff
@@ -28,6 +28,10 @@ def get_common_argument_parser(**flags):
         help='Path to a config file with task/model-specific parameters.',
         required=True)
 
+    parser.add_argument('--metrics-dump',
```

Comment:
metric_dump argument has not to be used in the train.py of the segmentation example. Thus, I propose to make it optional by adding it to the argument parser using the add_argument function. You should also modify the train.py of the segmentation example so that metric_dump is not used.